clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -43,7 +43,8 @@
     "@aspnet/signalr": "^1.0.0-preview3"
   },
   "dependencies": {
-    "msgpack5": "^4.0.2"
+    "msgpack5": "^4.0.2",
+    "tslib": "^1.9.3"
```

Comment:
FYI @Eilon @davidfowl @muratg  : new dependency on both `signalr` and `signalr-protocol-msgpack`. This is a library shipped by the TypeScript team that contains all the helpers they would normally inject into a compiled JS file. This is a recommended way to use the helpers as it allows multiple libraries to share the helpers in a single app rather than having the same helpers injected into every single file.

Unlike other dependencies we've added so far, this is a **runtime dependency** and not just for build. We can chat on Monday if you want to discuss it.

As a precedent: [`@angular/core`](https://www.npmjs.com/package/@angular/core) has this same dependency

Reply:
Lets discuss monday